### PR TITLE
Fix for crashes when using tinymce in STATICFILES_DIRS in development mode.

### PR DESCRIPTION
**Pull request by Felix-neko**
Here I've added some quick and dirty fix when using STATICFILES_DIRS instead of STATIC_ROOT.

### CODE REVIEW

File: tinymce/settings.py [hunk 1]
```diff
@@ -13,9 +13,12 @@
 
 if 'staticfiles' in settings.INSTALLED_APPS or 'django.contrib.staticfiles' in settings.INSTALLED_APPS:
     JS_URL = getattr(settings, 'TINYMCE_JS_URL',os.path.join(settings.STATIC_URL, 'tiny_mce/tiny_mce.js'))
-    JS_ROOT = getattr(settings, 'TINYMCE_JS_ROOT',os.path.join(settings.STATIC_ROOT, 'tiny_mce'))
+    JS_ROOT = getattr(settings, 'TINYMCE_JS_ROOT', None)
```
Comment (Natim) [inline tinymce/settings.py]: I think we should use the static library to build the right PATH here.

Comment (Felix-neko) [inline tinymce/settings.py]: Static library? What do you mean?

*scratching head*

**Comment by Natim** [comment]
Actually it doesn't really makes sense to use STATICDIRS without a STATIC_ROOT because collectstatic will copy files from STATICDIRS to STATIC_ROOT

**Comment by Felix-neko** [comment]
Alas, it does make sense when we use the debug server.

File: tinymce/settings.py (hunk 1 above)
Comment (Natim) [inline tinymce/settings.py]: It is what powers the findstatic command: https://docs.djangoproject.com/en/1.8/ref/contrib/staticfiles/#findstatic

Comment (Natim) [inline tinymce/settings.py]: See https://github.com/django/django/blob/master/django/contrib/staticfiles/management/commands/findstatic.py#L22

**Comment by Natim** [comment]
> Alas, it does make sense when we use the debug server.

Yes true.

Then we should use:
```
from django.contrib.staticfiles import finders
finders.find('tiny_mce', all=False)
```

**Comment by Felix-neko** [comment]
And where should I copy-paste the code? To settings.py?

**Comment by Natim** [comment]
Here I suppose: https://github.com/aljosa/django-tinymce/pull/131/files#diff-1d5fa3b685a77f9452e28aa136cd8319L16

**Comment by Felix-neko** [comment]
Done.
^__^

**Comment by Natim** [comment]
I think it is something we want to do if staticfiles is present in the INSTALLED_APPS and not if we use MEDIA.

Also please remove the documentation changes.

**Comment by Felix-neko** [comment]
>> Also please remove the documentation changes.
Done.

>> I think it is something we want to do if staticfiles is present in the INSTALLED_APPS and not if we use MEDIA.

Alas, I don't understand what do you mean. Maybe that's because I'm not a pro Django coder. = (

**Comment by Natim** [comment]
Look at the diff, you should see that you removed ``os.path.join(settings.STATIC_ROOT, 'tiny_mce')`` please replace it with the ``finders.find()`` call, also place the import at the top of the file (in the alphabetical order)

**Comment by Felix-neko** [comment]
[hopefully]

Maybe you modify it yourself?

**Comment by Natim** [comment]
It's your PR and your fix isn't it?

**Comment by Felix-neko** [comment]
Okay, let's do it...

**Comment by Felix-neko** [comment]
Done. Is it good enough?

**Comment by Natim** [comment]
I will let line 19 as it was otherwise it looks to be good yes.

**Comment by Felix-neko** [comment]
Done!

**Comment by Natim** [comment]
Thanks a lot. It looks good for me.

**Comment by Felix-neko** [comment]
Thank you, comrade!
^____^

**Comment by Natim** [comment]
I just released 2.0.5 with your fix.

**Pull request merged**